Makefile

```diff
@@ -35,6 +35,11 @@ TEST_FLAGS ?=
 test: toitpkg $(GO_MOCKS)
 	tedi test -v -cover $(TEST_FLAGS) ./...
 
+.PHONY: update-gold
+update-gold: export UPDATE_PKG_GOLD = true
```

Comment:
Is this better than

```
UPDATE_PKG_GOLD=true $(MAKE) test
```

Reply:
It does exactly the same.

Reply:
Maybe it's better on Windows, but I'm not sure.